dags/hivemind_etl_helpers/tests/integration/test_gdrive_loader.py

```
@@ -255,3 +280,119 @@ def test__load_by_drive_id(self, mock_load_data):
+
+        self.assertEqual(len(transformed_docs), 3)
+        self.assertEqual(
+            transformed_docs[0].metadata,
+            {
+                "file id": "file_1",
+                "author": "author_1",
+                "file name": "file_name_1",
+                "mime type": "mime",
+                "created at": "date",
+                "modified at": "modified",
+                "url": f"https://drive.google.com/file/d/file_1/view",
```

Ruff (0.8.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)